test/invariant/fuzz/properties/PropertiesAllo.t.sol

```diff
@@ -1,66 +1,345 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.19;
 
-import {HandlersParent} from '../handlers/HandlersParent.t.sol';
+import {HandlersParent} from "../handlers/HandlersParent.t.sol";
+import {IAllo, Allo, Metadata} from "contracts/core/Allo.sol";
+import {IRegistry, Registry} from "contracts/core/Registry.sol";
 
 contract PropertiesAllo is HandlersParent {
+    ///@custom:property-id 1
+    ///@custom:property one should always be able to pull/push correct (based on strategy) allocation for recipient
 
-///@custom:property-id 1
-///@custom:proprty one should always be able to pull/push correct (based on strategy) allocation for recipient                       
+    ///@custom:property-id 2
+    ///@custom:property a token allocation never “disappears” (withdraw cannot impact an allocation)
 
-///@custom:property-id 2
-///@custom:proprty a token allocation never “disappears” (withdraw cannot impact an allocation)                                      
+    ///@custom:property-id 3
+    ///@custom:property an address can only withdraw if has allocation
 
-///@custom:property-id 3
-///@custom:proprty an address can only withdraw if has allocation                                                                    
+    ///@custom:property-id 4
+    ///@custom:property profile owner can always create a pool
 
-///@custom:property-id 4
-///@custom:proprty profile owner can always create a pool                                                                            
+    ///@custom:property-id 5-a
+    ///@custom:property profile owner is the only one who can always add profile members (name ⇒ new anchor())
+    function prop_onlyProfileOwnerCanAddProfileMember(address _newMember) public {
+        // Get the profile ID
+        IRegistry.Profile memory _profile = registry.getProfileByAnchor(_ghost_anchorOf[msg.sender]);
 
-///@custom:property-id 5
-///@custom:proprty profile owner is the only one who can always add/remove/modify profile members (name ⇒ new anchor())              
+        address[] memory _members = new address[](1);
+        _members[0] = _newMember;
 
-///@custom:property-id 6
-///@custom:proprty profile owner is the only one who can always initiate a change of profile owner (2 steps)                         
+        (bool _success,) = targetCallDefault(
+            address(registry), 0, abi.encodeWithSelector(registry.addMembers.selector, _profile.id, _members)
```

Comment:
use abi.encodeCall, or you might revert due to type/prototype errors without Medusa telling you